platform/linux-generic/pktio/loop.c

```diff
@@ -339,6 +346,9 @@ static int loopback_send(pktio_entry_t *pktio_entry, int index ODP_UNUSED,
 	ret = odp_queue_enq_multi(queue, (odp_event_t *)hdr_tbl, nb_tx);
 
 	if (ret > 0) {
+		if (odp_unlikely(tx_ts_idx && ret >= tx_ts_idx))
```

Comment:
I think it would be better to have odp_unlikely() apply already to tx_ts_idx alone, i.e. if (odp_unlikely(tx_ts_idx) && ret >= tx_ts_idx).

Reply:
OK, will fix.